README.md

```diff
@@ -51,14 +37,6 @@ docker run -P stoplight/prism:3 mock -h 0.0.0.0 api.oas2.yml
 - [ ] Recording / "Learning" mode to create spec files
 - [ ] Data Persistence (allow Prism act like a sandbox)
 
-## Experimental programmatic APIs (advanced topic)
```

Comment:
We won't expose this API any more?

Reply:
It was a huge oversight on my part that this API ever made it in. We were not meant to be advertising the JS API until it was stable, and we were meant to be explaining it was experimental in the `packages/http/README.md` not the _main README_. Putting this here before it is ready (#511) is an advertisement we don't want.

Hopefully not too many people have started building anything in production around it.